/**:
  ros__parameters:
    duration: 40.0
    eps_diff: 1.0e-06
    q_init: 0.2
    timestep: 0.01
    urdf_path: ~/ros2_ws/dynamic_ws/src/vi/urdf/7_pendulum.urdf
atsvi_ad_node:
  ros__parameters:
    h_max: 0.1
    h_min: 1e-6
    max_adapt_iters: 100
etsvi_node:
  ros__parameters:
    lyap_alpha: 0.45
    lyap_beta: 0.005
etsvi_op_node:
  ros__parameters:
    pid_Kd: 1e-9
    pid_Ki: 1e-4
    pid_Kp: 0.0093
